src/pip/_internal/commands/show.py

```diff
@@ -60,6 +60,10 @@ def search_packages_info(query):
         installed[canonicalize_name(p.project_name)] = p
 
     query_names = [canonicalize_name(name) for name in query]
+    missing = [pkg for pkg in query_names if pkg not in installed]
+    if missing:
+        msg = 'WARNING: Package(s) not found: %s' % ', '.join(missing)
```

Comment:
pip's logging logic will prefix the "WARNING: " and logger.warn[ing] will format the message with % automatically. You can use the following instead:

```py
        logger.warn('Package(s) not found: %s', ', '.join(missing))
```

Reply:
Nice done as a part of the commit to also fix 2 lint errors in the tests.